src/NServiceBus.Transport.SQS.AcceptanceTests/When_trying_to_subscribe_when_topic_does_not_exist.cs

```diff
@@ -0,0 +1,123 @@
+
+            await Scenario.Define<Context>(c =>
+            {
+                c.MessageId = messageId;
+                c.EnableInstallers = true;
+                c.DisableAutoSubscribe = true;
+            })
+            .WithEndpoint<Endpoint>()
+            .Done(context => true)
+            .Run();
+
+            var context = await Scenario.Define<Context>(c =>
+            {
+                c.MessageId = messageId;
+                c.EnableInstallers = false;
+                c.DisableAutoSubscribe = false;
+            })
+            .WithEndpoint<Endpoint>()
+            .Done(context => true)
+            .Run();
+
+            Assert.IsTrue(context.Logs.Any(a => a.Message.Contains("not found") && a.Message.Contains("When_trying_to_subscribe_when_topic_does_not_exist-MyEvent")));
+        }
+
+        public class Context : ScenarioContext
+        {
+            public Guid MessageId { get; set; }
+            public bool EnableInstallers { get; set; }
+            public bool DisableAutoSubscribe { get; set; }
+        }
+
+        public class Endpoint : EndpointConfigurationBuilder
+        {
+            public Endpoint() =>
+                EndpointSetup<DefaultServerWithNoInstallers>((e, c) =>
+                {
+                    var ourContext = c.ScenarioContext as Context;
+                    if (ourContext.EnableInstallers)
+                    {
+                        e.EnableInstallers();
+                    }
+                    if (ourContext.DisableAutoSubscribe)
+                    {
+                        e.DisableFeature<AutoSubscribe>();
+                    }
+                });
+
+            class Handler : IHandleMessages<MyEvent>
+            {
+                public Task Handle(MyEvent message, IMessageHandlerContext context)
+                {
+                    return Task.CompletedTask;
+                }
+            }
+
+        }
+
+        public class MyMessage : ICommand
+        {
+        }
+
+        public class MyEvent : IEvent
+        {
+        }
+
+        public class DefaultServerWithNoInstallers : ServerWithInstallersDisabled
+        {
+            public IConfigureEndpointTestExecution PersistenceConfiguration { get; set; } = TestSuiteConstraints.Current.CreatePersistenceConfiguration();
+
+            public override Task<EndpointConfiguration> GetConfiguration(RunDescriptor runDescriptor, EndpointCustomizationConfiguration endpointConfiguration, Func<EndpointConfiguration, Task> configurationBuilderCustomization) =>
+                base.GetConfiguration(runDescriptor, endpointConfiguration, async configuration =>
+                {
+                    await configuration.DefinePersistence(PersistenceConfiguration, runDescriptor, endpointConfiguration);
+
+                    await configurationBuilderCustomization(configuration);
+                });
+        }
+
+        public class ServerWithInstallersDisabled : IEndpointSetupTemplate
```

Comment:
Let's move this out into a seperate file

Reply:
I have moved both of those into separate files and the relevant tests into a new folder 'Installers'